rules/dp_process.smk

```diff
@@ -0,0 +1,47 @@
+"""
+This workflow focuses in processing dp profiler features.
+"""
+
+
+configfile: "configs/configuration.yaml"
+
+
+rule dp_aggregate_profiles:
+    input:
+        index_file=INDEX_FILE,
+        dp_features_dir="data/{dp_name}_dp",
```

Comment:
is it possible to be slightly more specific with the variable `dp_name`? I'm not sure how to interpret what this actually represents.

Reply:
`dp_name` refers to the name of the directory that contains the deep profiler features. Would `deep_profiler_dir` sound better?

Reply:
yes, definitely!